docs/deployment.md

```diff
@@ -136,6 +136,36 @@ Zeit [Now](http://zeit.co/now) is a great service for running Probot apps. After
 
         $ now scale https://a-fancier-url.now.sh 1
 
+**ProTip™**
+
+Instead of passing the environment variables each time when deploying, you can set secrets using [`now secret add`](https://zeit.co/docs/features/env-and-secrets). In that case, you would do the following (make sure to replace `1234` and `mysecret` the values for your app)
+
+    $ now secret add app-id 1234
+    $ now secret add webhook-secret mysecret
+    $ now secret add private-key-base64 mysecret "$(cat ~/Downloads/*.private-key.pem | base64)"
```

Comment:
You can't do multiple secrets in a single invocation of `now secret add`?

Reply:
doesn’t look like it

```
$ now secret --help

  𝚫 now secrets [options] <command>

  Commands:

    ls                               Show all secrets in a list
    add      [name] [value]          Add a new secret
    rename   [old-name] [new-name]   Change the name of a secret
    rm       [name]                  Remove a secret
```

Reply:
Not sure how new this is, but I've been playing around with now recently and started using `now -E`

```sh
Options:
...
  -E FILE, --dotenv=FILE         Include env vars from .env file. Defaults to '.env'`
...
```

Since we use `.env` files in Probot, this seems like a pretty good option to recommend. All I have when I run `now secrets ls` is the base64 encoded key:

```shell
$ now secrets ls
> 1 secret found under tcbyrd [550ms]

  name         created
  private-key  9d ago
```

Then I run `now -E` to have it dynamically load the same secrets I use for local development from my `.env` file.

Any caveats on this method over the other? I've only done it with a test app, but it seems to be working ok.

Reply:
I deploy all my apps from Travis where I don’t have an .env file. I think we will be on the safe side by recommending to use secrets but we can also mention the method by passing env variables or a path to a .env file when deploying